internal/pkg/deploy/cloudformation/stack/transformers_test.go

```diff
@@ -411,8 +414,20 @@ func Test_convertAutoscaling(t *testing.T) {
 		badRange         = manifest.IntRangeBand("badRange")
 		mockRequests     = 1000
 		mockResponseTime = 512 * time.Millisecond
-		mockCPU          = manifest.Percentage(70)
-		mockMem          = manifest.Percentage(80)
+		perc             = manifest.Percentage(70)
+		timeMinute       = time.Second * 60
+		mockCPU          = manifest.ScalingConfigOrPercentage{
+			ScalingConfig: manifest.AdvancedScalingConfig{
+				Value: &perc,
+				Cooldown: &manifest.Cooldown{
```

Comment:
Are we considering the fact that The `cooldown` field can only be specified if `range` is specified somewhere?